package.json

```diff
@@ -42,11 +42,11 @@
 	},
 	"dependencies": {
 		"@beam-australia/react-env": "^3.1.1",
-		"@cosmjs/amino": "^0.27.1",
+		"@cosmjs/amino": "^0.29.4",
 		"@cosmjs/launchpad": "^0.27.1",
```

Comment:
this is the latest version already: https://www.npmjs.com/package/@cosmjs/launchpad